tests/e2e/chain/export.go

```diff
@@ -8,16 +8,21 @@ type ChainMeta struct {
 }
 
 type Validator struct {
-	Name          string `json:"name"`
-	ConfigDir     string `json:"configDir"`
-	Index         int    `json:"index"`
-	Mnemonic      string `json:"mnemonic"`
-	PublicAddress string `json:"publicAddress"`
+	Name           string `json:"name"`
+	ConfigDir      string `json:"configDir"`
+	Index          int    `json:"index"`
+	Mnemonic       string `json:"mnemonic"`
+	PublicAddress  string `json:"publicAddress"`
+	PublicAddress2 string `json:"publicAddress2"`
+	PublicKey      string `json:"publicKey"`
+	OperAddress    string `json:"operAddress"`
 }
 
 type Chain struct {
 	ChainMeta  ChainMeta    `json:"chainMeta"`
 	Validators []*Validator `json:"validators"`
+	PropNumber int          `json:"propNumber"`
```

Comment:
What's prop number?

Reply:
Prop Number keeps what the current prop number is. This way when we vote for the current prop, we can simply call this and then once it passes, increase this number by one. Please let me know if there is a better way for us to query what the most recent prop number is

Reply:
Why do we need this passed from chain initialization? We should minimize the changes to this struct as it is only used for genesis / initial state initialization.

For the purposes of superfluid, we can create proposals either in `e2e_setup.go` it in `TestSuperfluidVoting` to avoid modifying this struct